SECP-service/src/main/resources/assets/app/js/portal/filter-controller.js

```diff
@@ -0,0 +1,19 @@
+angular.module('SECP')
+    .controller('FilterController', ['$scope', function ($scope) {
+        var filtertable =$('#filtertable').DataTable({
+            //Need drawCallBack to take load select fields on every draw event of the table
+            drawCallback: function() {
+                $('.permission-level-select').select2();
+                $('.role-select').select2();
+            }
+        });
+
+        $('#filtertable tbody').on('click', 'button[data-target="#delete"]', function () {
```

Comment:
This line is a little bit complicated. You can just use angular ng-click.

Reply:
✔️